lib/js_locale_helper.rb

```diff
@@ -150,7 +150,8 @@ def self.output_MF(locale)
           )
         end
         .compact_blank
-    compiled = MessageFormat.compile(message_formats.keys, message_formats, strict: false)
+    js_message_formats = message_formats.transform_keys(&:dasherize)
```

Comment:
Here we transform the keys so they have a dash instead of an underscore because MessageFormat needs this. (so `fr-FR` instead of `fr_FR`)
We don’t modify the original hash, that way in the rescue block we don’t have to remember which version of the locale name to use.